meta_data.lean

```diff
@@ -132,13 +132,14 @@ do eltp ← to_expr tp,
    add_decl $ mk_definition meta_data_name []
                   `(unfinished_meta_data) eldt
 
-@[user_command]
-meta def unfinished_cmd (meta_info : decl_meta_info) (_ : parse $ tk "unfinished") : lean.parser unit :=
-do nm ← ident,
-   tk ":",
-   tp ← lean.parser.pexpr 0,
-   tk ":=",
-   struct ← lean.parser.pexpr,
-   add_unfinished nm tp struct
+--@[user_command]
```

Comment:
@picrin By the way, if you really want to skip `user_command` but only during CI, you have to find a reliable way to erase the definition of `user_command` in the testing script to account for foreseeable future changes to `meta_data.lean`. The current PR not only erase the definition of `user_command` during CI but also in the actual development. This is probably what @andrejbauer is trying to point out too.